src/db.rs

```diff
@@ -1448,23 +1469,32 @@ mod tests {
 			(vec![1; 953], key2, key3, vec![4; 79])
 		};
 
-		let mut inner_options = InternalOptions::default();
-		inner_options.create = true;
-		inner_options.commit_stages = db_test;
+		let inner_options =
+			InternalOptions { create: true, commit_stages: db_test, ..Default::default() };
 		let db = Db::open_inner(&options, &inner_options).unwrap();
 		assert_eq!(db.get(col_nb, &key1).unwrap(), None);
 
 		let mut iter = db.iter(col_nb).unwrap();
 		assert_eq!(iter.next().unwrap(), None);
+		assert_eq!(iter.prev().unwrap(), None);
 
 		db.commit(vec![(col_nb, key1.clone(), Some(b"value1".to_vec()))]).unwrap();
 		db_test.run_stages(&db);
 
 		assert_eq!(db.get(col_nb, &key1).unwrap(), Some(b"value1".to_vec()));
-		iter.seek(&[]).unwrap();
+		iter.seek_to_first().unwrap();
 		assert_eq!(iter.next().unwrap(), Some((key1.clone(), b"value1".to_vec())));
 		assert_eq!(iter.next().unwrap(), None);
 
+		iter.seek_to_first().unwrap();
+		assert_eq!(iter.next().unwrap(), Some((key1.clone(), b"value1".to_vec())));
+		assert_eq!(iter.prev().unwrap(), Some((key1.clone(), b"value1".to_vec())));
```

Comment:
@i1i1 question, shouldn't it be rather None?

Reply:
I believed that this is how it should behave, as after the call of `next` cursor was moved

Reply:
I may change it to absolute cursor in the future (not sure when), then it would return None instead. Is it fine with you if I switch to this behavior in a future pr ?

Reply:
It would work either way for me before. The main thing will be to change semantics before the release :)